airflow/contrib/hooks/gcp_vision_hook.py

```diff
@@ -435,14 +440,97 @@ def annotate_image(self, request, retry=None, timeout=None):
         """
         client = self.annotator_client
 
-        self.log.info('Annotating image')
+        self.log.info("Annotating image")
 
         response = client.annotate_image(request=request, retry=retry, timeout=timeout)
 
-        self.log.info('Image annotated')
+        self.log.info("Image annotated")
 
         return MessageToDict(response)
 
+    @GoogleCloudBaseHook.catch_http_exception
+    def text_detection(self, image, max_results=None, retry=None, timeout=None, additional_properties={}):
```

Comment:
https://docs.python-guide.org/writing/gotchas/ Dict is mutable

Reply:
Thanks, didn't know it works that way.
Changed it